.editorconfig

```diff
@@ -57,3 +60,20 @@ csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimenta
 # IDE0073: The file header is missing or not located at the top of the file
 file_header_template = Licensed to the .NET Foundation under one or more agreements.\nThe .NET Foundation licenses this file to you under the MIT license.
 dotnet_diagnostic.IDE0073.severity = error
+
+# CA1305: Specify IFormatProvider
+dotnet_diagnostic.CA1305.severity = none
```

Comment:
Runtime does this too.

If I don't do this, then all the `SR.Format` calls in this assembly (and only in this assembly) start complaining that I should use the method overload that takes an `IFormatProvider` argument.